maintainers/maintainer-list.nix

```diff
@@ -18253,6 +18253,12 @@
     githubId = 1108325;
     name = "Théo Zimmermann";
   };
+  znewman01 = {
```

Comment:
@znewman01 Do you still want to be maintainer?

Reply:
I'd like to be taken off, provided you have sufficient other maintainers. I've been a little too busy lately.

Thanks for picking this up!

Reply:
> provided you have sufficient other maintainers.

Unfortunately, we don't :(

Reply:
Nice, thank you!

Also, if you don't answer to notifications, nobody will blame you and you could still remove your self form the maintainer list when you want.